_includes/manuals/1.18/3.4_install_from_source.md

```diff
@@ -4,7 +4,7 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.0, 2.1, 2.2, 2.3 or 2.4
+* Ruby 2.3, 2.4, or 2.5
 * NodeJS 4.0 or newer
```

Comment:
I'm not sure about this. In Travis we only test with node 6 or higher. EPEL also has this version.

Reply:
Yeah, we should to raise this to 6.14 (the version currently in EPEL).